tools/build_container.sh

```diff
@@ -0,0 +1,23 @@
+curl -sL https://a01tools.blob.core.windows.net/droid/linux/a01droid -o tools/a01droid
+chmod +x tools/a01droid
+chmod +x tools/get_index
```

Comment:
This should not be required. The file can be check in git as `755`. If you do want to `chmod`, it should happen in the `Dockerfile`.